### PR TITLE
Start of v0.0.3 development

### PR DESCRIPTION
**Pull request by myungjoo**
Build scripts updated for v0.0.3-rc1 development.

1. Single-binary build is the default.
2. Cleaned up build scripts.
3. Being ready for Android. (will create a .deb for android development)

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -6,16 +6,19 @@ OPTION(TIZEN "Enable Tizen build mode" OFF)
...
+	MESSAGE(STATUS "The nnstreamer elements will be created as multple plugin binaries, one binary for one element. This is no more the standard method to release since nnstreamer 0.0.3.")
 ENDIF(SINGLE_BINARY)
 
-ADD_DEFINITIONS(-DVERSION="0.0.2")
+ADD_DEFINITIONS(-DVERSION="0.0.3")
 
 SET(PREFIX ${CMAKE_INSTALL_PREFIX})
 SET(EXEC_PREFIX "${PREFIX}/bin")
-SET(INCLUDEDIR "${PREFIX}/include/${PROJECT_NAME}")
+SET(INCLUDE_INSTALL_DIR "${PREFIX}/include")
```
Comment (leemgs) [inline CMakeLists.txt]: Looks good.

File: CMakeLists.txt [hunk 2]
```diff
@@ -138,10 +141,11 @@ ENDIF(NOT DISABLE_TENSORFLOW)
 LIST(LENGTH FILTER_DEP FILTER_DEP_LEN)
 
 IF (SINGLE_BINARY)
-	ADD_LIBRARY(nnstreamerStatic STATIC gst/nnstreamer/nnstreamer.c ${OBJECTS})
+	ADD_LIBRARY(nnstreamerStatic STATIC gst/nnstreamer/nnstreamer.c ${OBJECTS} gst/nnstreamer/tensor_common.c gst/nnstreamer/tensor_meta.c)
```
Comment (leemgs) [inline CMakeLists.txt]: It's strange. Do you want to generate a different file name for a static library as follows? 
- libnnstreamerStatic.a
- nnstreamer.so

The same file name style is better than a different file name structure between static and shared libary.

Comment (leemgs) [reply CMakeLists.txt]: I made a mistake. It's okay.

File: packaging/nnstreamer.spec [hunk 3]
```diff
@@ -166,5 +166,8 @@ install build/libcommon.a %{buildroot}%{_libdir}/
 
 
 %changelog
+* Mon Oct 15 2018 MyungJoo Ham <myungjoo.ham@samsung.com>
```
[the remark this answers is not in the document]
Comment (myungjoo) [reply packaging/nnstreamer.spec]: Added a commit to fix this.

File: debian/nnstreamer-dev.install [hunk 4]
```diff
@@ -1,3 +1,5 @@
 /usr/include/nnstreamer/*.h
 /usr/lib/pkgconfig/*.pc
-/usr/lib/*.a
+/usr/lib/libnnstreamer.a
```
[the remark this answers is not in the document]
Comment (myungjoo) [reply debian/nnstreamer-dev.install]: It already works as you wanted.

File: debian/changelog [hunk 5]
```diff
@@ -1,3 +1,9 @@
+nnstreamer (0.0.3-1rc1) stable; urgency=medium
```
Comment (leemgs) [inline debian/changelog]: :+1:

**Pull request merged**